cli/cmd/set_config.go

```diff
@@ -32,7 +32,7 @@ Supported keys:
 			cmd.SilenceUsage = false
 			return errors.New("required arguments KEY and VALUE")
 		}
-		configMng = config.NewCLIConfigManager()
+		configMng = config.NewCLIConfigManager("")
```

Comment:
Function signature `NewCLIConfigManager` has changed to support custom config file path. I have done a global replace on this function `NewCLIConfigManager()` -> `NewCLIConfigManager("")`